src/components/projectInfo/projectInfo.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import {  Row, Col, Image} from 'react-bootstrap'
+
+const ProjectInfo = () => {
+	return (
+		<Row className=" no-gutters bg-light position-relative align-items-center">
```

Comment:
niepotrzebna spacja po className="
```suggestion
		<Row className="no-gutters bg-light position-relative align-items-center">
```